javaslang/generator/Generator.scala

```diff
@@ -1311,6 +1347,10 @@ def generateMainClasses(): Unit = {
              */
             $Seq<?> toSeq();
 
+            <A> Tuple append(A v);
```

Comment:
Please add the additional field MAX_ARITY to the `Tuple` interface which might come handy in case a user wants to prevent UnsupportedOperationException on append/prepend without hard-coding the `this.arity() == ...` check:

```java
public interface Tuple {

    /**
     * The maximum arity of an Tuple.
     * <p>
     * Note: This value might be changed in a future version of Javaslang.
     * So it is recommended to use this constant instead of hardcoding the current maximum arity.
     */
    int MAX_ARITY = ${N};

    ...

}
```